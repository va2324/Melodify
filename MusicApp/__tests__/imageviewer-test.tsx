import React from 'react';
import ImageViewer from '@/components/ImageViewer';
import renderer from 'react-test-renderer';

describe('image viewer', () => {
    test('renders dog pic', () => {
        const tree = renderer.create(<ImageViewer imgSource={require('@/assets/images/dog.jpg')} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});