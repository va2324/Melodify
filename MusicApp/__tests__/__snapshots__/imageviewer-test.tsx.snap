// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image viewer renders dog pic 1`] = `
<ViewManagerAdapter_ExpoImage
  borderRadius={18}
  containerViewRef={"[React.ref]"}
  contentFit="cover"
  contentPosition={
    {
      "left": "50%",
      "top": "50%",
    }
  }
  height={440}
  nativeViewRef={"[React.ref]"}
  onError={[Function]}
  onLoad={[Function]}
  onLoadStart={[Function]}
  onProgress={[Function]}
  placeholder={[]}
  source={
    [
      {
        "testUri": "../../../assets/images/dog.jpg",
      },
    ]
  }
  style={
    {
      "borderRadius": 18,
      "height": 440,
      "width": 320,
    }
  }
  transition={null}
  width={320}
/>
`;
